libraries/core_libs/consensus/include/final_chain/final_chain.hpp

```diff
@@ -18,9 +18,11 @@ class FinalChain {
 
  protected:
   util::EventEmitter<std::shared_ptr<FinalizationResult>> const block_finalized_emitter_{};
+  util::EventEmitter<uint64_t> const block_applying_emitter_{};
```

Comment:
who is using this emitter ? I know what event it emits, but who and when needs it ?